app/jobs/sync_roles_and_variables.rb

```diff
@@ -0,0 +1,8 @@
+class SyncRolesAndVariables < ::ApplicationJob
```

Comment:
We should provide `humanized_name` so it appears in the tasks as human understandable task.
But :+1: for `ApplicationJob`